uber_sante/controllers/patient_controller.py

```diff
@@ -116,3 +117,27 @@ def patient():
             return js.create_json(None, "Email address already registered", js.ResponseReturnCode.CODE_500)
 
         return js.create_json(None, "Patient record created", js.ResponseReturnCode.CODE_201)
+
+    if request.method == 'DELETE':
```

Comment:
I think the "remove appointment" should have its own route, since the 'DELETE' in /patient should be for actually deleting a patient, not for deleting an appointment

Reply:
Yeah makes sense, Ill add a route /appointment